### DISCUSSION TITLE
SMB Share

### DISCUSSION THREAD

**Discussion by gswec2005** [IceWhaleTech/CasaOS#346] [Ideas] [answered]
For now i'm need to share my storage on network with samba, and experimenting webmin as interface.

there is a better option (with UI) or a plan to implement a widget to manage this function?

**Comment by JohnGuan** [comment, marked as answer]
Duplicate of #168

**Comment by gswec2005** [comment]
Hey, the other idea, is mount a NAS share to acces his files.

My idea here is for sharing files from CasaOS on. Network, to access from a tv or windows.
